packages/stark-ui/src/modules/date-range-picker/components/date-range-picker.component.ts

```diff
@@ -24,91 +39,185 @@ const componentName = "stark-date-range-picker";
-	@Input()
-	public endDateLabel = "STARK.DATE_RANGE_PICKER.TO";
+	@ViewChild("startPicker")
+	public startPicker!: StarkDatePickerComponent;
 
 	/**
-	 * Maximum date of the end date picker
+	 * Reference to the end datepicker embedded in this component
+	 */
+	@ViewChild("endPicker")
+	public endPicker!: StarkDatePickerComponent;
+
+	/*--- START-DATE CONFIGURATIONS ---*/
+	private _startBeforeEndValidator: ValidatorFn = ({ value }) => {
+		return value instanceof Date && this.endDate instanceof Date && value.getTime() > this.endDate.getTime()
+			? { startBeforeEnd: true }
+			: null;
+	};
+
+	/**
+	 * Source Date to be bound to the start datepicker model
 	 */
 	@Input()
-	public endMaxDate?: Date;
+	public get startDate(): Date | undefined {
+		return this._startDate.value || undefined;
+	}
+
+	public set startDate(value: Date | undefined) {
+		this._startDate.setValue(value);
+	}
 
 	/**
-	 * Minimum date of the end date picker
+	 * The internal formControl used to manage the state of the starkDatePicker for the start date.
+	 * This can be used to retrieve internal errors.
+	 *
+	 * @example
+	 *
+	 * <stark-date-range-picker required #starkDateRangePicker >
+	 *   <ng-container start-date-errors>
+	 *     {{starkDateRangePicker.startDateFormControl.hasError('required') ? "Start date is required" : null }}
+	 *   </ng-container>
+	 * </stark-date-range-picker>
+	 */
+	public get startDateFormControl(): FormControl {
+		return this._startDate;
+	}
+
+	/**
+	 * @ignore
+	 */
+	private _startDate = new FormControl();
+
+	/**
+	 * Label to be displayed in the end datepicker
 	 */
 	@Input()
-	public endMinDate?: Date;
+	public startDateLabel = "STARK.DATE_RANGE_PICKER.FROM";
 
 	/**
-	 * Whether the datepickers are disabled
+	 * Minimum date of the start date picker
 	 */
 	@Input()
-	public isDisabled?: boolean;
+	public startMinDate?: Date;
 
 	/**
-	 * HTML "name" attribute of the element.
+	 * Maximum date of the start date picker
 	 */
 	@Input()
-	public rangePickerId = "";
+	public startMaxDate?: Date;
+
+	/*--- END-DATE CONFIGURATIONS ---*/
+
+	private _endAfterStartValidator: ValidatorFn = ({ value }) => {
+		return value instanceof Date && this.startDate instanceof Date && value.getTime() < this.startDate.getTime()
+			? { endAfterStart: true }
+			: null;
+	};
 
 	/**
-	 * HTML "name" attribute of the element.
+	 * Source Date to be bound to the end datepicker model
 	 */
 	@Input()
-	public rangePickerName = "";
+	public get endDate(): Date | undefined {
+		return this._endDate.value || undefined;
+	}
+
+	public set endDate(value: Date | undefined) {
+		this._endDate.setValue(value || null);
+	}
 
 	/**
-	 * Source Date to be bound to the start datepicker model
+	 * The internal formControl used to manage the state of the starkDatePicker for the start date.
+	 * This can be used to retrieve internal errors.
+	 *
+	 * @example
+	 *
+	 * <end-date-range-picker required #endDateRangePicker >
+	 *   <ng-container start-date-errors>
```

Comment:
This line should be: `<ng-container end-date-errors>`

Reply:
🤦‍♂